module2/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+#include <cstdint>
```

Comment:
Same here cstdint should be included alphabetically with the rest of inlcudes.

Reply:
+1